lib/active_merchant/billing/gateways/adyen.rb

```diff
@@ -9,6 +9,7 @@ class AdyenGateway < Gateway
 
       self.supported_countries = ['AT', 'AU', 'BE', 'BG', 'BR', 'CH', 'CY', 'CZ', 'DE', 'DK', 'EE', 'ES', 'FI', 'FR', 'GB', 'GI', 'GR', 'HK', 'HU', 'IE', 'IS', 'IT', 'LI', 'LT', 'LU', 'LV', 'MC', 'MT', 'MX', 'NL', 'NO', 'PL', 'PT', 'RO', 'SE', 'SG', 'SK', 'SI', 'US']
       self.default_currency = 'USD'
+      self.currencies_without_fractions = %w(CVE DJF GNF IDR JPY KMF KRW PYG RWF UGX VND VUV XAF XOF XPF)
```

Comment:
FYI @jknipp @curiousepic I added a local override for currencies_without_fractions according to  https://docs.adyen.com/development-resources/currency-codes/#description and added a CLP test to confirm intended behavior.